test/Directory.Packages.props

```diff
@@ -0,0 +1,138 @@
+<Project>
+  <Import Project="..\Directory.Packages.props" />
+
+  <!-- TODO: I'm having second thought on whether it would be better to consolidate all packages in one ItemGroup and sort them alphabetically. -->
```

Comment:
I recommend one ItemGroup with all packages sorted in order.

Reply:
Done.